src/main/java/teammates/storage/api/CoursesDb.java

```diff
@@ -153,4 +165,14 @@ private Course getCourseEntity(String courseId) {
     
         return courseList.get(0);
     }
+    
+    private List<Course> getCourseEntities(List<String> courseIds) {
+        Query q = getPM().newQuery(Course.class);
+        q.setFilter(":p.contains(ID)");
```

Comment:
Hmmm the jar has no attached Javadoc, so not sure what I should point it to. Also, this type of filter (`:p.contains(paramName)`) has already been used in a few other Dbs.